purchase_order_product_summary/report/purchase_order_summary_template.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_purchasesummary_document">
+        <t t-call="web.external_layout">
+            <t t-set="o" t-value="o.with_context(lang=o.partner_id.lang)"/>
+            <div class="page" id="product_summary">
+                <div class="oe_structure"/>
+                <div class="row justify-content-start">
```

Comment:
> I am missing something of the header: purchase order and date of the order, for example. I don't think the whole header of the detailed format is needed.

Product summary was implemented as an independent template, so we need to add again those fields. Then, no problem adding the fields we're interested in. This works:

```suggestion
                <div class="row justify-content-start">
                <h2 t-if="o.state == 'draft'">Request for Quotation #<span t-field="o.name"/></h2>
                <h2 t-if="o.state in ['sent', 'to approve']">Purchase Order #<span t-field="o.name"/></h2>
                
                <div id="summary_informations" class="row mt32 mb32">
                  <div t-if="o.date_order" class="col-3 bm-2">
                      <strong>Order Date:</strong>
                      <p t-field="o.date_order" class="m-0"/>
                  </div>
                </div>
```